climpyrical/spytialProcess.py

```diff
@@ -0,0 +1,123 @@
+from collections import OrderedDict
+
+from nptyping import NDArray
+from typing import Any, Tuple
+import numpy as np
+from rpy2.robjects.packages import importr
+from rpy2.robjects import FloatVector, DataFrame
+from rpy2 import robjects
+
+importr("fields")
+
+
+def fit(
+    latlon: NDArray[(2, Any), float],
+    z: NDArray[(Any,), float],
+    nx: int,
+    ny: int,
+    xy: Tuple,
```

Comment:
Yup, 

```python
xy: Tuple[int, int]
```
Although `typing` does not work with `isinstance` (unlike how `nptyping` works) which is fine since I want remove my hard checking in favour of `mypy` anyway.